taskcluster/ci/toolchain/android.yml

```diff
@@ -0,0 +1,31 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+job-defaults:
+    run:
+        using: toolchain-script
+    treeherder:
+        kind: build
+        platform: toolchains/opt
+        tier: 1
+    worker-type: b-android
+    worker:
+        docker-image: {in-tree: base}
+        max-run-time: 1800
+
+
+linux64-android-sdk-linux-repack:
+    attributes:
+        artifact_prefix: mobile/android-sdk
```

Comment:
Is this needed?

Reply:
You're right, it's not! I removed it.

Reply:
Oh, it turns out to be needed! Otherwise, the sdk ends up in the `public/build` folder: https://firefox-ci-tc.services.mozilla.com/tasks/H7VhhRRXQZuhtk1c0ytlCQ#artifacts